### PR TITLE
IPv6 dafault to IPv4 methods in gossip is_global 

### PR DESCRIPTION
**Pull request by danielSanchezQ**
I run into this while reading the code. I think we can rely on the IPv4 methods if the IPv6 is transformable. At least until we have the IPv6 ones.

### CODE REVIEW

**Comment by danielSanchezQ** [comment]
I do not know if this is being tested. I'm searching for a place where I can write some tests for it. It would be nice if someone points out for me :)

**Comment by mzabaluev** [comment]
@danielSanchezQ Feel free to add unit tests to the module.

File: jormungandr/src/network/p2p/gossip.rs [hunk 1]
```diff
@@ -72,9 +72,20 @@ impl Gossip {
                 if ip.is_loopback() {
                     return false;
                 }
-                // FIXME: add more tests when Ipv6Addr convenience methods
-                // get stabilized:
+                // Try using same methods by trying to cast address to ipv4
+                // FIXME: use Ipv6 tests when Ipv6Addr convenience methods are get stabilized:
```
Comment (NicolasDP) [inline jormungandr/src/network/p2p/gossip.rs]:
```suggestion
                // FIXME: use Ipv6 tests when Ipv6Addr convenience methods get stabilized:
```

**Comment by danielSanchezQ** [comment]
Thanks @mzabaluev

**Pull request merged**